addons/mail/static/src/models/activity/activity.js

```diff
@@ -225,6 +225,10 @@ registerModel({
         },
     },
     fields: {
+        activityView: one2many('mail.activity_view', {
```

Comment:
For clarity, plural on this field since it's `x2many`. So `activityViews`

Reply:
comment still open, should be plural